src/main/java/net/dries007/tfc/ConfigTFC.java

```diff
@@ -69,6 +69,10 @@ public static void onConfigChangedEvent(ConfigChangedEvent.OnConfigChangedEvent
         @Config.LangKey("config." + MOD_ID + ".general.leafStickDropChanceBonusClasses")
         public String[] leafStickDropChanceBonusClasses = new String[] {"knife", "scythe"};
 
+        @Config.Comment("List of fluids allowed to be picked up by wooden bucket")
+        @Config.LangKey("config." + MOD_ID + ".general.woodenBucketWhitelist")
+        public String[] woodenBucketWhitelist = new String[] {"fresh_water", "hot_water", "salt_water", "water", "limewater", "tannin", "olive_oil", "vinegar", "rum", "beer", "whiskey", "rye_whiskey", "corn_whiskey", "sake", "vodka", "cider", "brine", "milk_curdled", "milk_vinegar"};
```

Comment:
Oh, and as of b7c9c2e6a105d91e56d6c359925ea2099113538c I'm going to be picky about this: Please add an `en_us.lang` entry with the lang key and tooltip of any config values you add. See the existing ones for style - the name should be a readable version of the variable name and the tooltip should match the comment.